data/onPostBuild/transpileMdxToMarkdown.test.ts

```
@@ -58,6 +59,66 @@ Content without intro`;
+      const input = `---
+title: Test Page
+---
+
+Some content here.`;
+
+      const navContext: NavContext = {
+        prev: { name: 'Previous', url: 'https://ably.com/docs/prev.md', description: 'Previous page.' },
+        next: { name: 'Next', url: 'https://ably.com/docs/next.md', description: 'Next page.' },
+        siblings: [
+          { name: 'Sibling', url: 'https://ably.com/docs/sibling.md', description: 'A sibling page.' },
+        ],
```

ESLint

[error] Replace `⏎··········{·name:·'Sibling',·url:·'https://ably.com/docs/sibling.md',·description:·'A·sibling·page.'·},⏎········` with `{·name:·'Sibling',·url:·'https://ably.com/docs/sibling.md',·description:·'A·sibling·page.'·}`

(prettier/prettier)